pants.toml

```diff
@@ -389,6 +389,7 @@ pytest_plugins.add = [
   # TODO(#8651): We need this until we switch to implicit namespace packages so that pytest-cov
   #  understands our __init__ files. NB: this version matches 3rdparty/python/requirements.txt.
   "setuptools==40.6.3",
+  "ipdb",
```

Comment:
Rather than baking this in, should we take advantage of CLI args and make it a pure docs change that suggests `--pytest-plugins=ipdb` ?

Reply:
We could, but that increases the barrier to being able to use ipdb. I spend most my time in debuggers and I'd personally love to avoid having to type `--pytest-pytest-plugins=ipdb --pytest-args='s'` every time. It's already a bummer that we have to have `--pytest-args='s'`.

Given that this solely impacts us Pants devs, I'm hoping that it's okay to add this permanently.

Reply:
Hm. It doesn't seem like it solely affects Pants devs.

> It's already a bummer that we have to have --pytest-args='s'.

Mmm... I'll get a PR out to remove that deprecation today.

Reply:
> Hm. It doesn't seem like it solely affects Pants devs, but ok!

What do you mean? We're only installing this in our internal `pants.toml` so this won't impact any other users. 

Do you mean that we should update the docs so that other users know how to use idpb in their own code base? We could, but this is `howto_develop.md` so I don't expect many people to be reading that.

Reply:
I'm suggesting that Pants devs are not the only ones who might want to:
1. add additional deps to a run
2. use passthrough args

If there is a lot of boilerplate in doing those kinds of things, that affects all users (or they won't figure out how to in the first place).

> Do you mean that we should update the docs so that other users know how to use idpb in their own code base? We could, but this is howto_develop.md so I don't expect many people to be reading that.

I'm ok with adding it here. I was just referencing that "adding a debugger" is a thing that users are likely to want to do as well. Maybe that means it's worth calling out on our end-user docs, but I don't personally know how common `ipdb` usage is vs `pdb`, so have no idea whether it makes a useful example.